airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -74,6 +79,7 @@ def __init__(self, job_name, job_definition, job_queue, overrides, max_retries=4
         self.job_definition = job_definition
         self.job_queue = job_queue
         self.overrides = overrides
+        self.parameters = parameters
```

Comment:
Really happy this is being added! Would it make sense to name the keyword arg `job_parameters` to better differentiate it from the `params` keyword arg?